code/ui/components/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Badge } from './Badge';
```

Comment:
The following stories in this PR have just been moved, and many of them have been converted from `storiesOf` to CSF3.

The diff shows completely new content, not really true.